src/main/java/rest/SearchResource.java

```diff
@@ -177,29 +178,24 @@ public PersonDTO_OUT addPersonWithHobbies(PersonDTO_IN person) {
                         content = @Content(mediaType = "application/json", schema = @Schema(implementation = PersonDTO_OUT.class))),
                 @ApiResponse(responseCode = "200", description = "The Requested Person"),
                 @ApiResponse(responseCode = "404", description = "Person not found")})
-
     public PersonDTO_OUT getPersonInfoByPhone(@QueryParam("phone") long phone) {
         //    /api/search/phone?phone=<phone>
-        //get from facade
-        return new PersonDTO_OUT();
+        return FACADE.getPersonByPhone(phone);
     }
 
     @GET
     @Path("/city")
     @Produces(MediaType.APPLICATION_JSON)
     @Operation(summary = "Get all persons living in a given city (i.e. 2800 Lyngby)",
-            tags = {"General"},
+            tags = {"General"}, parameters = @Parameter(array = @ArraySchema(schema = @Schema(implementation = CityInfoDTO_IN.class))),
             responses = {
                 @ApiResponse(
                         content = @Content(mediaType = "application/json", schema = @Schema(implementation = CityInfoDTO_OUT.class))),
                 @ApiResponse(responseCode = "200", description = "The Requested list of persons"),
                 @ApiResponse(responseCode = "404", description = "No persons in that city found")})
-    public List<PersonDTO_OUT> getPersonsByCity(@QueryParam("city") String city) {
-        //    /api/search/city?city=<city>
-        //example mentions both zip & city?
-        //get from facade
-        List<PersonDTO_OUT> returnList = new ArrayList();
-        return returnList;
+    public List<PersonDTO_OUT> getPersonsByCity(@QueryParam("zip") String zip, @QueryParam("city") String city) {
+        //    /api/search/city?zip=<zip>&city=<city>
+        return FACADE.getPersonsInCity(new CityInfoDTO_IN(zip, city));
```

Comment:
Sick that it just worked that smoothly. I was highly doubtful that my methods would work.